FirebaseAdmin/FirebaseAdmin/Messaging/FirebaseMessagingClient.cs

```diff
@@ -239,6 +234,20 @@ private async Task<HttpResponseMessage> SendRequestAsync(object body, Cancellati
             return batch;
         }
 
+        private async Task<FirebaseMessagingException> CreateExceptionFor(HttpResponseMessage response)
```

Comment:
That 'For' at the end of the name is really weird in C#.

Reply:
Removed.